openjdk/build.xml

```diff
@@ -324,14 +324,16 @@
 			</else>
 		</if>
 		<checkGitRepoSha repoDir="openjdk-jdk" />
+		<addTestenvProperties repoDir="openjdk-jdk" repoName="JDK${env.JDK_VERSION}_${env.JDK_IMPL^^}"/>
```

Comment:
Please remove extra `^^`

Reply:
The "^^" is used to force the JDK_IMPL variable to be uppercase